setup.py

```diff
@@ -34,7 +34,8 @@ def read(fname):
         'rdflib',
```

Comment:
Can you please add here the same rdflib version declaration that stwfsapy uses? Currently it [seems to be](https://github.com/zbw/stwfsapy/blob/master/setup.py) `rdflib>=4.2,<6.0`
This way we can avoid version conflicts, for example when rdflib 6.0 is eventually released.